scripts/apps/search/directives/SearchContainer.ts

```diff
@@ -28,6 +36,18 @@ export function SearchContainer() {
                         }
                     }
                 });
+                preferencesService.get().then((result) => {
+                    if (result !== undefined && FILTERS_PANEL_PREFERENCES_KEY in result) {
```

Comment:
if result is null then `null !== undefined` will be `true`. We use this convention to check bot null and undefined: `(result != null)`

Reply:
done